filters/json.js

```diff
@@ -0,0 +1,62 @@
+// i18next j4 json format defined at https://www.i18next.com/misc/json-format
+import flatten from "flat";
+
+export class JsonFilter {
+    constructor(params) {
```

Comment:
`{ enableARBAnnotations, enablePluralSuffixes, emitComments }` is more readable than `params` and self-documenting without having to read the rest of the code

Reply:
I get an error `TypeError: Cannot destructure property 'enableArbAnnotations' of 'undefined' as it is undefined.` with this syntax.